scripts/caclmgrd

```diff
@@ -115,6 +116,8 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
     DualToR = False
     bfdAllowed = False
+    VxlanAllowed = False
```

Comment:
@dgsudharsan  Is this VxlanAllowed variable used?

Reply:
Yes. It is used in line number 992